### PR TITLE
Adds a tip about ERP and More

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request
What does ERP stand for?
And how to help people that are board and can be helping sci!

## Why It's Good For The Game
People /need/ to know these tips

## Changelog
:cl:
add: more silly tips
/:cl:

### CODE REVIEW

**Comment by KathrinBailey** [comment]
"If your ever board"

"If you're ever bored, and don't want to hangout with people, just ask Sci for a Tetris board and make yourself useful well not playing SS13!"

Hah, I get to bully people now!

File: strings/sillytips.txt [hunk 1]
```diff
@@ -45,3 +45,5 @@ Help.
 Maints.
 BZ stops or slows down Lings chem regeneration drastically, make sure to BZ flood the station when lings are confirmed!
 Admins always regret meme options in their polls.
+ERP stands for Emergent Role Play.
+If your ever board, and don't want to hangout with people, just ask Sci for a Tetris board and make yourself usefull well not playing SS13!
```
Comment (dapnee) [inline strings/sillytips.txt]:
```suggestion
If you’re ever bored and don't want to hangout with people, just ask science for a Tetris board and make yourself useful while not playing SS13!
```

**Comment by Trilbyspaceclone** [comment]
That was not every Emergent Grammar of you

**Comment by silicons** [comment]
tetris isn't even fullmerged =|

**Comment by Linzolle** [comment]
tetris is gone lol, remove that line or i'll close this soon

File: strings/sillytips.txt [hunk 2]
```diff
@@ -45,3 +45,12 @@ Help.
 Maints.
 BZ stops or slows down Lings chem regeneration drastically, make sure to BZ flood the station when lings are confirmed!
 Admins always regret meme options in their polls.
+ERP stands for Emergent Role Play.
+Remember if you see a bug report it on Github.
+Have good ideas for the game? Well make a suggestion on Github, we have templates just for it!
+Role Play is fun. Try it sometimes.
+This is a MRP Server, no really.
+Clean bot has more to say after being emaged. Try it sometimes with your limited use Emags... It will be worth it!
+Bar sign can be emaged.
```
Comment (Linzolle) [inline strings/sillytips.txt:54]: Pretty sure it would be spelled "emagged" but it's a made up word so who knows.

File: strings/sillytips.txt [hunk 3]
```diff
@@ -45,3 +45,12 @@ Help.
...
+Mood.
+Ok Boomer, is not a helpful thing to say when being bowinked about your wrong doing.
```
Comment (Linzolle) [inline strings/sillytips.txt:56]: Bwoinked, though I'm not sure about this one either.

Comment (Trilbyspaceclone) [reply strings/sillytips.txt:56]: Bowinked, as its the sound of a bow like a cross/bow/, and a ink like pen and /ink/?

File: strings/sillytips.txt (hunk 2 above)
Comment (Trilbyspaceclone) [reply strings/sillytips.txt:54]: e like the letter
mag like a /mag/net right?
so it should be 1 g?

File: strings/sillytips.txt (hunk 3 above)
Comment (Linzolle) [reply strings/sillytips.txt:56]: what

File: strings/sillytips.txt (hunk 2 above)
Comment (Linzolle) [reply strings/sillytips.txt:54]: pretty sure the past tense of every verb that ends in a G has two G's. Lugged, sagged, hugged, gagged, etc.

**Comment by Ghommie** [comment]
The R&D Testris computer has been un-testmerged as the tetris relied on client-side scripts (and thus easier to exploit) and had an issue that allowed people to farm research points by pausing the game repeatedly. Also the ERP joke is actually not quite funny.

**Pull request closed without merging**